diesel/src/pg/metadata_lookup.rs

```diff
@@ -47,7 +47,21 @@ where
                     .store_type(cache_key, type_metadata);
                 PgTypeMetadata(Ok(type_metadata))
             }
-            Err(_e) => PgTypeMetadata(Err(FailedToLookupTypeError::new(cache_key.into_owned()))),
+            Err(_e) => {
+                let metadata_cache = self.get_metadata_cache();
+                let lower_schema = schema.map(|schema| schema.to_lowercase()).unwrap();
```

Comment:
We shouldn't call `unwrap()` here as the schema name could be empty. Just doing the following things should be fine:
```suggestion
                let lower_schema = schema.map(|schema| schema.to_lowercase());
```

This will transform the schema name to lower case if there is a schema name, otherwise it will just set `lower_schema` to `None`